tools/spec-gen-sdk/src/automation/workflow.ts

```diff
@@ -49,7 +48,7 @@ export type WorkflowContext = SdkAutoContext & {
   messageCaptureTransport: Transport;
   scriptEnvs: { [key: string]: string | undefined };
   tmpFolder: string;
-  extraResultRecords: MessageRecord[];
```

Comment:
Is `extraResultRecords` used somewhere else to push the logs?

Reply:
The `extraResultRecords` variable only uses an empty array and does not modify any data in this code.